### ISSUE TITLE
Opacity is not included with generated icon. Having opacity will help to include duotone fonts. Is there any way?

### ISSUE THREAD

**Issue by nemaanshul** [tancredi/fantasticon#144]

**Comment by nemaanshul** [comment]
@tancredi

**Comment by tancredi** [comment]
Hi @nemaanshul - Fonts only contain vector glyphs with no alpha / color data - if you need alpha layers or multiple colours you should use SVG images. Make sense?

**Comment by nemaanshul** [comment]
Great @tancredi, thanks!

**Comment by oneezy** [comment]
> Hi @nemaanshul - Fonts only contain vector glyphs with no alpha / color data - if you need alpha layers or multiple colours you should use SVG images. Make sense?

hi @tancredi , can you describe how **material design icons** do it? They support 2-tone font icons
https://fonts.google.com/icons

![image](https://user-images.githubusercontent.com/7369575/141367601-fb4e6c0f-eba6-4b55-b363-c8e14e233d55.png)

**Comment by oneezy** [comment]
@tancredi 
I also have a use case where I need the original `fill` color from the svg icon as someone else requested in #133. 

I maintain a [Cryptocurrency Font](https://github.com/ScavengerBot/CryptoFont) (400+ icons) and rely on the `fill` color to represent the coins original branding. I too use icomoon to generate icons but it would be _Fantastic_ if we could make this a part of our build step instead 😄 

I'd be happy to create a new issue for tracking purposes if you'd want to make this a feature.

https://github.com/ScavengerBot/CryptoFont

![image](https://user-images.githubusercontent.com/7369575/141369427-84359310-450d-4274-b978-6760f7355468.png)

**Comment by oscargm** [comment]
Looks like font-awesome is splitting the icon in 2 different icons and overlapping them using `::before` and `::after` pseudo-elements as mentioned [here](https://stackoverflow.com/questions/69364961/how-can-i-change-the-color-of-one-portion-of-a-font-icon).
If you open [this font awesome page](https://fontawesome.com/v6.0/docs/web/style/duotone) and inspect an icon and select the pseudo-elements you'll see that they do the same.

**Comment by robincsamuel** [comment]
Thanks for the pointer @oscargm!

I've generated duotone icons by splitting the SVG into two files so that two different glyphs are generated. I then modified the CSS to change the second icon definition to make it `::after`.

*Example*:

*Input: home-dual.svg*

- Modify the CSS templates to extend the `::before` rules for `::after` as well and add a rule to position the dual icons' `::before` absolute. Also, set opacity to `0.3` or so.
- Split the svg and generate `home-dual.svg` with one section, and then `home-dual-layer2.svg` with only the second section.
- Generate the icons, and then replace all occurrences of `-layer2:before` with `:after` in the CSS file (I'm trying to figure out if I can do this using the `hbs` templates).
